contents/docs/integrations/js-integration.mdx

```diff
@@ -147,7 +149,12 @@ You can also pass two more arguments to `posthog.identify`. Both take objects co
 
 You can read more about the difference between this [in the previous section](#setting-user-properties-via-an-event).
 
-Warning! You can't call identify straight after an .init (as .init sends a page view, probably with the user's anonymous id. To combat this, you can call .init with a callback, inside which you can then call identify.
+<blockquote class="warning-note">
+    Warning! You can't call identify straight after an `.init` (as `.init` sends a page view, probably with the user's
+    anonymous ID.
+</blockquote>
+
+To address the issue described above, you can call `.init` with a callback, inside which you can then call identify,
```

Comment:
"To address the issue described above, you can call `.init` passing a `loaded` callback function, inside which you can then call `identify`, like so":